CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @bbishopMSFT @ridhoq @sajayantony
```

Comment:
I have a team called @Azure/obom-contributors. We can use that here or create a new group called obom-maintainers and use that here. Thoughts?

Reply:
Most projects follow explicit owners.  Keeps is transparent and also reviewable. GitHub teams hides the addition or removal of people. 

But it’s good for notification and mentioning in issues.